### ISSUE TITLE
Uncaught TypeError: Cannot read property 'fn' of undefined bootstrap-slider.js:358

### ISSUE THREAD

**Issue by bradintheusa** [seiyria/angular-bootstrap-slider#13]
Hi

i am banging my head on this error.during load:

`Uncaught TypeError: Cannot read property 'fn' of undefined bootstrap-slider.js:358`

i know it is in my code as the samples work fine.  I was hoping someone else may have seen/solved the issue before I slowly take my app apart.

Thanks

**Comment by sheymann** [comment]
Hi @seiyria, bootstrap-slider is v4.5.1 now and seems stabilized. Any update to remove the reference to jQuery plugin slider?

Thanks

**Comment by seiyria** [comment]
No, there has not been a patch to do that as far as I know. You're welcome to submit a PR to do it though!
